packages/fx-core/src/plugins/resource/keyvault/v3/index.ts

```diff
@@ -0,0 +1,100 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { hooks } from "@feathersjs/hooks/lib";
+import {
+  AzureSolutionSettings,
+  err,
+  FxError,
+  Inputs,
+  ok,
+  Result,
+  v2,
+  v3,
+} from "@microsoft/teamsfx-api";
+import * as path from "path";
+import { Service } from "typedi";
+import { ArmTemplateResult } from "../../../../common/armInterface";
+import { Bicep } from "../../../../common/constants";
+import { generateBicepFromFile } from "../../../../common/tools";
+import { CommonErrorHandlerMW } from "../../../../core/middleware/CommonErrorHandlerMW";
+import { getTemplatesFolder } from "../../../../folder";
+import { AzureResourceKeyVault } from "../../../solution/fx-solution/question";
+import { BuiltInFeaturePluginNames } from "../../../solution/fx-solution/v3/constants";
+import { Constants } from "../constants";
+
+@Service(BuiltInFeaturePluginNames.keyVault)
+export class KeyVaultPluginV3 implements v3.FeaturePlugin {
```

Comment:
Just to confirm: No 'activate' function in v3, right?

Reply:
Yes, all plugins are added by addFeature command